src/hotspot/os_cpu/bsd_aarch64/os_bsd_aarch64.cpp

```diff
@@ -756,6 +819,10 @@ int os::extra_bang_size_in_bytes() {
   return 0;
 }
 
+void os::current_thread_enable_wx_impl(WXMode mode) {
+  pthread_jit_write_protect_np(mode != WXWrite);
```

Comment:
might as well set argument type explicitly
For example argument could be
mode == WXExec ? true : false